generators/action/index.js

```diff
@@ -99,9 +113,16 @@ module.exports = class extends Generator {
 
     this.log('Creating action file');
     mkdirp('lib/actions');
-    this.fs.copy(
-      this.templatePath('action.js'),
+    const actionFile = actionType;
+    this.log(actionFile);
+
+    this.fs.copyTpl(
+      this.templatePath(`${actionFile}.js`),
       this.destinationPath(`lib/actions/${id}.js`),
+      (() => {
+        if (mType === 'Static') return { metadata: '' };
+        return { metadata: 'exports.getMetaModel = async function getMetaModel(cfg) {};' };
```

Comment:
will the static option leave 2 newlines at the end of templates? if so, can you delete 1 newline in the templates and change the dynamic option to return `exports.getMetaModel = ...` along with 1 newline?